Added github-test script

xbankov opened the pull request:
Added test step to workflows

xbankov commented:
@Witiko , do you understand this error? I did nothing with the previous files.

xbankov commented:
@Witiko How did you resolve this issue with previous scripts? I do not see any special chmod or any permission change in code. Thanks

xbankov commented:
> ou resolve this issue with previous scripts? I do not see any special chmod or any permission change in code. Thanks

Resolved, `git add -chmod=x`

Witiko commented:
> @Witiko How did you resolve this issue with previous scripts? I do not see any special chmod or any permission change in code. Thanks

Git tracks file permissions. Running `chmod +x file` and `git add file` locally should work on UN*X platforms.

Witiko commented:
The build error is due to the `set -o nounset` on top of the bash script, which requires that all variables are defined before first use:

> Treat unset variables and parameters other than the special parameters "@" and "*" as an  error  when  performing parameter expansion.  If expansion is attempted on an unset variable or parameter, the shell prints an error message, and, if not interactive, exits with a non-zero status. — An excerpt from `man bash`

xbankov commented:
@Witiko to sum up:
- I added test step to both workflows
- I created the test script
- I added second binary ffprobe, which is required by test, I do not think it could increase docker size dramatically
- I hopefully resolved issue fit gihub-configure to always declare env variable

Witiko commented:
Job well done, thanks for the merge.
